docs/guides/rule_based_profilers/how-to-create-a-new-expectation-suite-using-rule-based-profilers.md

```diff
@@ -0,0 +1,114 @@
+---
+title: How to create a new Expectation Suite using Rule Based Profilers
+---
+import Prerequisites from '../connecting_to_your_data/components/prerequisites.jsx';
+
+<Prerequisites>
+
+- Have configured a Datasource to connect to your data
+- Have a basic understanding of [Metrics in Great Expectations](https://docs.greatexpectations.io/en/latest/reference/core_concepts/metrics.html)
+- Have a basic understanding of [Expectation Configurations in Great Expectations](https://docs.greatexpectations.io/en/latest/reference/core_concepts/expectations/expectations.html#expectation-concepts-domain-and-success-keys)
+
+</Prerequisites>
+
+
+### Profilers
+
+A Profiler uses an Execution Engine to build a new Expectation Suite. It can use zero, one, or more Batches of data to decide which Expectations to include in the new Suite. It can use other information, like a schema or an idea. A profiler may be used to create basic high-level expectations based on a schema even without data, to create specific Expectations based on team conventions or statistical properties in a dataset, or even to generate Expectation Suites specifically designed to be rendered by a Descriptive Renderer for data discovery.
```

Comment:
Might it make sense to highlight GE-specific terms in some manner? Either using hyperlinks or `code formatting` to signify to the user that they need to have an understanding of the term before moving forward?

If I'm reading this paragraph and come across "Execution Engine", a term I'm not super familiar with, having the ability to easily navigate to the definition could be very valuable. I think highlighting or hyperlinking every instance of a term would be a bit much but maybe just the first instance? I've read documentation that has on-hover CSS to create definition pop-ups on specific keywords (that's very much not the scope of this PR but that's what inspired these thoughts).

You do explicitly state that you need to know x, y, and z before using this tutorial so there's a strong likelihood that users will be coming into this with the requisite understanding.

Reply:
This is a great point. I lifted this definition from our existing docs, but agree that it may be too jargony too early. I'll modify it  (though I'm also hoping that users follow the pre-requisites).